### PR TITLE
docs: improved the en version of tutorial_sql_1

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
fixed links
fixed the format of titles
fixed typos in zh version

### CODE REVIEW

File: docs/en/tutorial/tutorial_sql_1.md [hunk 1]
```diff
@@ -1,43 +1,52 @@
...
+For example, for the real-time recommendation system, the original data may only be the keywords entered by users, such as "washing machine", and the corresponding user and commodity data tables stored in the database. 
+In order to make better real-time recommendation, the following features may be more helpful:
 
-- The home appliance brand that the user purchased the most in the past year
-- The average consumption level of the user in the past three years
-- In the past hour, the discount on the platform has been more than 70%, which is in line with the top three washing machine models purchased by users in this user gender and age group
+- The home appliance brand that the user purchased the most in the past year.
+- The average consumption level of the user in the past three years.
+- The top three kinds washing machines purchased most by clients of the same gender and the same age group as the user, and were over 30% off in the past one hour.
```
Comment (lumianph) [inline docs/en/tutorial/tutorial_sql_1.md]: The top three kinds washing machines
The top three models of washing machines

File: docs/en/tutorial/tutorial_sql_1.md [hunk 2]
```diff
@@ -1,43 +1,52 @@
...
 
-- It is recommended to use docker image to run this tutorial under the standalone version. Refer to [OpenMLDB Quick Start](https://openmldb.ai/docs/en/main/quickstart/openmldb_quickstart.html) for image pull and CLI operation mode. If using the cluster version, please use the offline mode (`SET @@execute_mode='offline'` ). The normal online mode under the cluster version's CLI only supports the simple data preview function, so most of the SQL in the tutorial cannot run.
-- All data related to this tutorial can be downloaded [here]( https://openmldb.ai/download/tutorial_sql/tutoral_sql_data.zip)
+- It is recommended to use docker image to run this tutorial under the standalone version. For image pulling and CLI operation methods, please refer to [OpenMLDB Quick Start](../quickstart/openmldb_quickstart.md). If you want to use the cluster version, please use the offline mode (`SET @@execute_mode='offline'` ). The CLI of cluster version only supports online preview mode and offline mode. And the online preview mode only supports the simple data preview function, so most of the SQL in the tutorial cannot run in the online preview mode.
```
Comment (lumianph) [inline docs/en/tutorial/tutorial_sql_1.md]: use the docker image

File: docs/en/tutorial/tutorial_sql_1.md [hunk 3]
```diff
@@ -52,32 +61,37 @@ Note that if you want to run the SQL illustrated in this tutorial, please follow
...
+In addition to the primary table, there may also be tables storing relevant auxiliary information in the database, which can be spliced with the primary table through the JOIN operation. These tables are called **Secondary Tables** (note that there may be multiple secondary tables). For example, we can have a secondary table storing the merchants' history flow. In the process of feature engineering, more valuable information can be obtained by splicing the primary and secondary tables. The feature engineering over multiple tables will be introduced in detail in the  [next part](tutorial_sql_2.md) of this series.
```
Comment (lumianph) [inline docs/en/tutorial/tutorial_sql_1.md]: spliced 
combined

Comment (lumianph) [reply docs/en/tutorial/tutorial_sql_1.md]: splicing the primary
combining the ...

File: docs/en/tutorial/tutorial_sql_1.md [hunk 4]
```diff
@@ -52,32 +61,37 @@ Note that if you want to run the SQL illustrated in this tutorial, please follow
...
+Before discussing the details of feature extraction, we can categorize the features commonly used in machine learning. There are four common features in machine learning according to feature datasets' building and data aggregation methods:
 
 - Single-row features on the primary table: Computing expressions and functions for one or more columns on the primary table.
-- Time-series features on the primary table: Building sliding time-series windows for the primary table and performing aggregate functions over the windows.
-- Single-row features on multiple tables: The primary table joins the secondary tables, and then it performs the single-row features on the joined table.
-- Time-series features on multiple tables: A row of the primary table matches multiple rows from a secondary table, and then it performs time-series features on the matched rows.
+- Time-series features on the primary table: Building sliding time-series windows for the primary table and extracting time-series features over the windows.
```
Comment (lumianph) [inline docs/en/tutorial/tutorial_sql_1.md]: sliding time-series windows 
sliding windows

File: docs/en/tutorial/tutorial_sql_1.md [hunk 5]
```diff
@@ -86,96 +100,96 @@ hour(trans_time) as f_trans_hour,
...
+The `w1d` window shown in the above figure is for the row which `id=9`, and the `w1d` window contains three rows (`id=6`, `id=8`, `id=9`). These three rows fall in the time window [2022-02-07 12:00:00, 2022-02-08 12:00:00] .
 
+- w1d_10d: the window from 1 day ago to the last 10 days
 ```sql
--- The windows from 1 day ago to the last 10 days
 window w1d_10d as (PARTITION BY uid ORDER BY trans_time ROWS_RANGE BETWEEN 10d PRECEDING AND 1d PRECEDING)
 ```
 
-The window `w1d_10d` for the row with `id=9` contains three rows, which are with `id=1`, `id=3` and `id=4`. These three rows fall within the time window of [2022-01-29 12:00:00, 2022-02-07 12:00:00]。
+The window `w1d_10d` for the row which `id=9` contains three rows, which are `id=1`, `id=3` and `id=4`. These three rows fall in the time window of [2022-01-29 12:00:00, 2022-02-07 12:00:00]。
```
Comment (lumianph) [inline docs/en/tutorial/tutorial_sql_1.md]: The window `w1d_10d` for the row which `id=9`
The window `w1d_10d` for the partition `id=9`

File: docs/en/tutorial/tutorial_sql_1.md [hunk 6]
```diff
@@ -289,23 +302,23 @@ window w30d as (PARTITION BY uid ORDER BY trans_time ROWS_RANGE BETWEEN 30d PREC
 
 **Frequency Statistics**
 
-We make frequency statistics for a given column. 
+We make frequency statistics for a given column as we may need to know the type of the highest frequency, the proportion of the type with the largest number, etc., in each category.
```
Comment (lumianph) [inline docs/en/tutorial/tutorial_sql_1.md:305]: ratio might be better than proportion

File: docs/en/tutorial/tutorial_sql_1.md [hunk 7]
```diff
@@ -289,23 +302,23 @@ window w30d as (PARTITION BY uid ORDER BY trans_time ROWS_RANGE BETWEEN 30d PREC
...
 
-`fz_top1_ratio`：Find the ratio of the maximum count of a category in the window to the total count of the window.
+`fz_top1_ratio`: Find out the type with the largest number and compute the proportion of its number in the window.
```
Comment (lumianph) [inline docs/en/tutorial/tutorial_sql_1.md:307]: ratio

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/en/tutorial/tutorial_sql_1.md (hunk 6 above)
Comment (michelle-qinqin) [reply docs/en/tutorial/tutorial_sql_1.md:305]: the ratio of men to women in this department  stands at 4:6.

the proportion of women in this department is 60%.

so `ratio` maybe more suitable if it is compared with the type with the second largest number?

Comment (lumianph) [reply docs/en/tutorial/tutorial_sql_1.md:305]: i see, then proportion is fine

**Review by lumianph — APPROVED** [review]

**Pull request merged**